src/cluster/slot_import.cc

```diff
@@ -21,57 +21,60 @@
 #include "slot_import.h"
 
 SlotImport::SlotImport(Server *srv)
-    : Database(srv->storage, kDefaultNamespace), srv_(srv), import_slot_(-1), import_status_(kImportNone) {
+    : Database(srv->storage, kDefaultNamespace), srv_(srv), import_slot_range_(-1, -1), import_status_(kImportNone) {
   std::lock_guard<std::mutex> guard(mutex_);
   // Let metadata_cf_handle_ be nullptr, then get them in real time while use them.
   // See comments in SlotMigrator::SlotMigrator for detailed reason.
   metadata_cf_handle_ = nullptr;
 }
 
-Status SlotImport::Start(int slot) {
+Status SlotImport::Start(const SlotRange &slot_range) {
   std::lock_guard<std::mutex> guard(mutex_);
   if (import_status_ == kImportStart) {
     // return ok if the same slot is importing
-    if (import_slot_ == slot) {
+    if (import_slot_range_ == slot_range) {
       return Status::OK();
     }
-    return {Status::NotOK, fmt::format("only one importing slot is allowed, current slot is: {}", import_slot_)};
+    return {Status::NotOK,
+            fmt::format("only one importing slot is allowed, current slot is: {}", import_slot_range_.String())};
   }
 
   // Clean slot data first
-  auto s = ClearKeysOfSlot(namespace_, slot);
+  auto s = ClearKeysOfSlotRange(namespace_, slot_range);
   if (!s.ok()) {
     return {Status::NotOK, fmt::format("clear keys of slot error: {}", s.ToString())};
   }
 
   import_status_ = kImportStart;
-  import_slot_ = slot;
+  import_slot_range_ = slot_range;
   return Status::OK();
 }
 
-Status SlotImport::Success(int slot) {
+Status SlotImport::Success(const SlotRange &slot_range) {
   std::lock_guard<std::mutex> guard(mutex_);
-  if (import_slot_ != slot) {
-    return {Status::NotOK, fmt::format("mismatch slot, importing slot: {}, but got: {}", import_slot_, slot)};
+  if (import_slot_range_ != slot_range) {
+    return {Status::NotOK, fmt::format("mismatch slot, importing slot: {}, but got: {}", import_slot_range_.String(),
+                                       slot_range.String())};
   }
 
-  Status s = srv_->cluster->SetSlotImported(import_slot_);
+  Status s = srv_->cluster->SetSlotRangeImported(import_slot_range_);
   if (!s.IsOK()) {
-    return {Status::NotOK, fmt::format("unable to set imported status: {}", slot)};
+    return {Status::NotOK, fmt::format("unable to set imported status: {}", slot_range.String())};
   }
 
   import_status_ = kImportSuccess;
   return Status::OK();
 }
 
-Status SlotImport::Fail(int slot) {
+Status SlotImport::Fail(const SlotRange &slot_range) {
   std::lock_guard<std::mutex> guard(mutex_);
-  if (import_slot_ != slot) {
-    return {Status::NotOK, fmt::format("mismatch slot, importing slot: {}, but got: {}", import_slot_, slot)};
+  if (import_slot_range_ != slot_range) {
+    return {Status::NotOK, fmt::format("mismatch slot, importing slot: {}, but got: {}", import_slot_range_.String(),
```

Comment:
```suggestion
    return {Status::NotOK, fmt::format("mismatch slots, importing slots: {}, but got: {}", import_slot_range_.String(),
```

Reply:
Thanks for the advice. Actually, if you search for `".String()"`, there are many similar issues.
![image](https://github.com/apache/kvrocks/assets/46661603/b5af819b-b026-448c-91fa-2d01bf3f365c)

In the beginning, I tried changing "slot" to "slots", but encountered some difficulties. On one hand, when `start == end`, `String()` will return a single `slot` instead of `[start,end]`, which causes ambiguity with the term "slots" in the message. On the other hand, some expected values in the test cases also need to be changed from "slot" to "slots". So I temporarily abandoned these modifications.

```cpp
std::string String() const {
  if (!IsValid()) return "-1";
  if (start == end) return fmt::format("{}", start);  // <== HERE
  return fmt::format("{}-{}", start, end);
}
```

Regarding the ambiguity issue, we could probably change it to "slot(s)" instead of "slot" or "slots" to be as compatible as possible with both representations. If you think that's OK, I'll change them (including some cases in the test cases).

Reply:
Thanks for the deep thought. If you think it is not trouble, better modify it :)